numpy/lib/function_base.py

```diff
@@ -764,6 +764,22 @@ def copy(a, order='K', subok=False):
     >>> x[0] == z[0]
     False
 
+    Note that ``np.copy`` is a shallow copy and will not
+    copy object elements within arrays:
+
+    >>> obj = dict()
+    >>> a = np.array([obj], dtype=object)
+    >>> b = np.copy(a)
+    >>> b[0] is a[0]
+    True
+
+    To copy the elements as well, use `copy.deepcopy`:
```

Comment:
```suggestion
    To ensure all elements within an ``object`` array are copied,
    use `copy.deepcopy`::
```
Wording suggestion again emphasizing `object` arrays.